main/src/cgeo/geocaching/connector/IConnector.java

```diff
@@ -52,6 +53,21 @@
     @Nullable
     String getLongCacheUrl(@NonNull Geocache cache);
 
+    /**
+     * Get the browser URL for the given LogEntry. May return null if no url aravilable or identifiable.
+     */
+    @Nullable
+    String getCacheLogUrl(@NonNull Geocache cache, @NonNull LogEntry logEntry);
+
+    /**
+     * For a given service-log-id (as assigned by this IConnector and stored as service log id in log entries),
+     * returns a logid ready for display to the user (e.g. in the GUI)
+     * May return null if there is no displayable log id.
+     */
+    @Nullable
+    String getDisplayableLogId(@Nullable String serviceLogId);
```

Comment:
Why would you need to display log ID to the user at all?

Reply:
I currently display it in the popup when you click on log as well as next to the log entry in the log list (as announced in the screenshot in the issue #8897 ). Don't like it? Let's discuss it there.

Reply:
I don't think log ID gives any value to the user :)

![image](https://user-images.githubusercontent.com/918446/94286698-a55f2300-ff55-11ea-868f-dd2a9358be13.png)

Reply:
@okainov I think it is better to have these discussions in the issue itself , because imho they are documented better that way. Following this issue, I wanted to delete the log ids on the log entries and change the title to "Log (logid) of (username)". But maybe you want to suggest something different?